src/domain/safe/entities/__tests__/multisig-transaction.builder.ts

```diff
@@ -9,6 +9,8 @@ import {
 } from './multisig-transaction-confirmation.builder';
 import { dataDecodedBuilder } from '../../../data-decoder/entities/__tests__/data-decoded.builder';
 
+const HASH_LENGTH = 10;
```

Comment:
I've experienced some flaky tests due to not having enough entropy in hashes generation. When calling `faker.string.hexadecimal` with no length, [it defaults to length 1](https://fakerjs.dev/api/string.html#hexadecimal), so it causes collisions sometimes and therefore makes [this check](https://github.com/safe-global/safe-client-gateway-nest/pull/556/files#diff-42037b77e31a8190067c1f5363cd928174ec81d75a0cc5ead485d17f5712369bR118) fail.